src/test/ui/issues/issue-40883.rs

```diff
@@ -78,8 +78,8 @@ fn verify_stack_usage(before_ptr: *mut Vec<Big>) {
     let stack_usage = isize::abs(
         (&mut stack_var as *mut _ as isize) -
             (before_ptr as isize)) as usize;
-    // give space for 2 copies of `Big` + 128 "misc" bytes.
-    if stack_usage > mem::size_of::<Big>() * 2 + 128 {
+    // give space for 2 copies of `Big` + 256 "misc" bytes.
+    if stack_usage > mem::size_of::<Big>() * 2 + 256 {
```

Comment:
Could this be binary searched so it's less than double the previous value?

Reply:
I thought powers of two were nice, but sure. ;)

Reply:
208 is the smallest value that works for me.
But I am a bit worried it might be larger for other platforms / architectures.

Reply:
We don't support a target with larger sizes/alignments than x64 I don't think.

Reply:
Oh nevermind, they could less stack efficient even if the types aren't larger.

Reply:
Yeah that's what I meant.